examples/dup_var.no

```diff
@@ -0,0 +1,13 @@
+fn main(pub public_input: Field, private_input: [Field; 3]) {
```

Comment:
please add a test in the `tests/examples.rs` to point to this test code.

Reply:
Added the test .